app/views/application/personal-information/residency.njk

```diff
@@ -12,11 +12,14 @@
 {% block primary %}
 
 <p class="govuk-body">To get help with student visas and your immigration status, speak to a <a href="https://beta-adviser-getintoteaching.education.gov.uk/">Get Into Teaching</a> adviser.</p>
+<h2 class="govuk-heading-m">Getting a visa through your training provider</h2>
+<p class="govuk-body">You may be able to get a visa through your teacher training provider. This is called sponsoring a student visa.</p>
```

Comment:
Does "through" imply that the teacher training provider will get the visa for you? (Whereas I think you still have to do the application yourself?)